t/run_after_spec.t

```diff
@@ -0,0 +1,57 @@
+use Test::Spec;
+my($first_nested_after, $second_nested_after, $inner_after, $second_inner_after, $todo_after);
+
+describe "outer describe block" => sub {
+    describe "inner block" => sub {
+        describe "first nested block" => sub {
+            it "runs first" => sub {
+                ok(! $first_nested_after ,"first nested block");
+            };
+            after all => sub {
+                $first_nested_after = 1;
+                ok(1, "after all - first nested block");
+            };
+        };
+        describe "second nested block" => sub {
+            it "first nested after has run by now" => sub {
+                ok($first_nested_after, "first nested after has run");
+            };
+            it "doesn't run second nest after until all it's have run" => sub {
+                ok($second_nested_after, "second nested after hasn't run yet")
```

Comment:
Commit 90080db7d7d1aad6ddb7da9898d8b8697aef6e72 should correct this error.  There was a flaw in the logic when describes had multiple examples.